### PR TITLE
Update table of contents in main README

### PR DESCRIPTION
**Pull request by emackey**
TOC updated using [markdown-toc extension for VSCode](https://marketplace.visualstudio.com/items?itemName=AlanWalk.markdown-toc).

Reorganized a small section that contained the VSCode editor.

### CODE REVIEW

**Comment by emackey** [comment]
Renamed that section and another one that re-used the term "Viewer".  Added Unreal link.

File: README.md [hunk 1]
```diff
@@ -198,6 +198,7 @@ To compare WebGL-based glTF loaders, see [gltf-test](https://github.com/cx20/glt
 | Tool | Status | Description |
 |------|--------|-------------|
 | [UnityGLTF](https://github.com/KhronosGroup/UnityGLTF) | ![status](https://img.shields.io/badge/glTF-2%2E0-green.svg?style=flat) | Unity3D library for exporting, loading, parsing, and rendering glTF assets |
+| [Unreal](https://forums.unrealengine.com/development-discussion/content-creation/1415582-the-new-gltf-import-supported-in-4-19-preview-is-awesome-experimentation) | ![status](https://img.shields.io/badge/glTF-2%2E0-green.svg?style=flat) | Unreal adds glTF import capability |
```
Comment (donmccurdy) [inline README.md]: Here are some direct links:

- https://docs.unrealengine.com/en-us/Studio/Unreal-Datasmith/Datasmith-Supported-Software-and-File-Types#formatssupportedbyunrealenginedirectly
- https://docs.unrealengine.com/en-us/Builds/4_19

But the forum thread does also seem helpful, with Mike chiming in there.

Comment (emackey) [reply README.md]: Done

**Comment by emackey** [comment]
Wait... hold.. spotted yet another problem

**Comment by emackey** [comment]
Ready.

**Comment by emackey** [comment]
This is ready for merge.

**Pull request merged**